### DISCUSSION TITLE
[Feature Request] Anchor Component

### DISCUSSION THREAD

**Discussion by Eating-Eating** [element-plus/element-plus#1522] [Ideas]
issue url:
https://github.com/element-plus/element-plus/issues/1149

描述:
问题主已经2个月没动静了，如果没人接手的话我想接。

实现：
两个组件：el-anchor锚点跟el-anchor-link锚点链接组件，还需建立一个scrollTo的公共动画方法，与el-backtop的动画差不多，后面可以将el-backtop中的scrollTop动画改为引用此方法。
一期基本功能：指定滚动的容器/锚点改变时事件/click事件覆盖。

样式：
hove字体变色，anchor左侧轴参考timline时间线中样式，primary主题色

Api:
anchor api: container/change/click
link api: href/target/title

refer to https://ant.design/components/anchor/

**Comment by yimingsir** [comment]
这个组件还会加入吗

**Comment by shawDirectUser** [comment]
请问这个有进度不？

**Comment by Fuphoenixes** [comment]
已添加
